stubs/jsonschema/jsonschema/protocols.pyi

```diff
@@ -19,4 +19,4 @@ class Validator(Protocol):
     def is_valid(self, instance: dict[Any, Any]) -> bool: ...
     def iter_errors(self, instance: dict[Any, Any]) -> Iterator[ValidationError]: ...
     def validate(self, instance: dict[Any, Any]) -> None: ...
-    def evolve(self, **kwargs) -> "Validator": ...
+    def evolve(self, **kwargs) -> Validator: ...
```

Comment:
I didn't use Self here because the implementation explicitly uses `"Validator"`.